salt/backups/backup_tracking.sls

```diff
@@ -1,9 +1,28 @@
 {% set edx_tracking_bucket = 'odl-residential-tracking-backup' %}
-{% set edx_tracking_bucket_backup_key = salt.pillar.get('backups:mitx_residential_tracking:aws_key') %}
-{% set edx_tracking_bucket_backup_keyid = salt.pillar.get('backups:mitx_residential_tracking:aws_keyid') %}
 {% set edx_tracking_local_folder = '/edx/var/log/tracking' %}
 {% set instance_id = salt.grains.get('id') %}
 
+ensure_tracking_bucket_exists:
+  boto_s3_bucket.present:
+    - Bucket: {{ edx_tracking_bucket }}
+    - region: us-east-1
+
+ensure_instance_profile_exists_for_tracking:
+  boto_iam_role.present:
+    - name: edx-instance-role
+    - delete_policies: False
+    - policies:
+        edx-old-tracking-logs-policy:
+          Statement:
+            - Action:
+                - s3:*
```

Comment:
This should be scoped to only those capabilities necessary to upload the files (you can refer to the vault role settings for that)

Reply:
https://github.mit.edu/mitx-devops/salt-pillar/blob/master/vault/init.sls#L124